tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_HwenoModifiedSolution.cpp

```diff
@@ -0,0 +1,1290 @@
+  //                      c24, c25, c26, c27, c28, c29, c30, c31, c32, c33, c34,
+  //                      c35], -2, 0, 0] - quad334[uLowerXi, -2, 0, 0])^2
+  //     + (quad334[trial[c0, c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12,
+  //                      c13, c14, c15, c16, c17, c18, c19, c20, c21, c22, c23,
+  //                      c24, c25, c26, c27, c28, c29, c30, c31, c32, c33, c34,
+  //                      c35], 2, 0, 0] - quad334[uUpperXi, 2, 0, 0])^2
+  //     + (quad334[trial[c0, c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12,
+  //                      c13, c14, c15, c16, c17, c18, c19, c20, c21, c22, c23,
+  //                      c24, c25, c26, c27, c28, c29, c30, c31, c32, c33, c34,
+  //                      c35], 0, 2, 0] - quad334[uUpperEta, 0, 2, 0])^2
+  //     + (quad334[trial[c0, c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12,
+  //                      c13, c14, c15, c16, c17, c18, c19, c20, c21, c22, c23,
+  //                      c24, c25, c26, c27, c28, c29, c30, c31, c32, c33, c34,
+  //                      c35], 0, 0, -2] - quad334[uLowerZeta, 0, 0, -2])^2,
+  //     quad334[trial[c0, c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12,
+  //                   c13, c14, c15, c16, c17, c18, c19, c20, c21, c22, c23,
+  //                   c24, c25, c26, c27, c28, c29, c30, c31, c32, c33, c34,
+  //                   c35], 0, 0, 0] == quad334[uLocal, 0, 0, 0],
+  // {c0, c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14,
+  //  c15, c16, c17, c18, c19, c20, c21, c22, c23, c24, c25, c26, c27,
+  //  c28, c29, c30, c31, c32, c33, c34, c35}
+  // ]
+  const auto expected = [&logical_coords]() noexcept {
+    const auto& x = get<0>(logical_coords);
+    const auto& y = get<1>(logical_coords);
+    const auto& z = get<2>(logical_coords);
+    const double c0 = 7734190419582420551. / 62148673763526035437.;
+    const double c1 = 765121. / 1263364.;
+    const double c2 = 786240. / 1895041.;
+    const double c3 = 124800. / 1105441.;
+    const double c4 = 0.;
+    const double c5 = 0.;
+    const double c6 = 374400. / 1105441.;
+    const double c7 = 0.;
+    const double c8 = 0.;
+    const double c9 = 308447321809657439079. / 621486737635260354370.;
+    const double c10 = -892944. / 1579205.;
+    const double c11 = -6250608. / 9475205.;
+    const double c12 = -878879. / 11054410.;
+    const double c13 = 0.;
+    const double c14 = 0.;
+    const double c15 = -595296. / 1105441.;
+    const double c16 = 1. / 2.;
+    const double c17 = 0.;
+    const double c18 = 21514562148445593021. / 124297347527052070874.;
+    const double c19 = 89424. / 315841.;
+    const double c20 = 625968. / 1895041.;
+    const double c21 = 99360. / 1105441.;
+    const double c22 = 0.;
+    const double c23 = 0.;
+    const double c24 = 298080. / 1105441.;
+    const double c25 = 0.;
+    const double c26 = 0.;
+    const double c27 = 3655223967127444271. / 310743368817630177185.;
```

Comment:
oh my. (not a change request, though I think some of these exceed the precision of the `double` epsilon...)